big5/test_procedures/ppl/ppl-schedule.json

```diff
@@ -0,0 +1,287 @@
+{
+  "operation": "ppl-default",
+  "warmup-iterations": {{ warmup_iterations | default(200) | tojson }},
+  "iterations": {{ test_iterations | default(100) | tojson }},
+  "target-throughput": {{ target_throughput | default(2) | tojson }},
+  "clients": {{ search_clients | default(1) }}
+},
+{
```

Comment:
Same as [this PR comment](https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/685#discussion_r2289440324). Can we modify this to ensure that params are configurable for specific queries.